standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -589,6 +589,9 @@ public static ConfVars getMetaConf(String name) {
             "in which a warning is being raised if multiple worker version are detected.\n" +
             "The setting has no effect if the metastore.metrics.enabled is disabled \n" +
             "or the metastore.acidmetrics.thread.on is turned off."),
+    COMPACTOR_METADATA_CACHE_TIMEOUT("metastore.compactor.metadata.cache.timeout",
+      "hive.metastore.compactor.metadata.cache.timeout", 60, TimeUnit.SECONDS,
```

Comment:
Might be too small. If we have a query running more than a min, then we will not cross-cache between Initiator and Cleaner

Reply:
The idea was to keep it at a low value so that we don't need extra eviction logic. Now that the cache works for a single cycle of the initiator, the sole purpose of this config is to be able to control the memory consumption of the cache.

Reply:
do we still need a timeout config as we currently programmatically evict the cache after each loop?